package.json

```diff
@@ -9,5 +9,12 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.1.2",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
```

Comment:
I wonder whether these were added automatically. If so, then maybe we do need to keep these in.

Reply:
Yeah they were all added automatically. I'll keep them in going forward